include/gatekeeper_net.h

```diff
@@ -30,6 +30,16 @@
 
 #include "gatekeeper_flow.h"
 
+/*
+ * Custom log type for Gatekeeper-related log entries
+ * that are not relevant to a specific block.
+ */
+extern int gatekeeper_logtype;
+
+#define G_LOG(level, ...)		\
+	rte_log(RTE_LOG_ ## level,	\
+	gatekeeper_logtype, "GATEKEEPER: " __VA_ARGS__)
```

Comment:
After the change of initialization of `gatekeeper_logtype` (see another comment), these definitions here should go to `include/gatekeeper_main.h`. Review the `#include "gatekeeper_net.h"` that this patch was adding.